docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -122,5 +130,130 @@ The following CIS benchmark checks cannot be automated and must be addressed man
 - 3.7 Audit Software Inventory
 - 6.2.1 Ensure Protect Mail Activity in Mail Is Enabled
 
+## Windows 10 Enterprise benchmark
+
+Fleet's policies have been written against v1.12.0 of the benchmark. You can refer to the [CIS website](https://www.cisecurity.org/cis-benchmarks) for full details about this version.
+
+### Checks that require a Group Policy Template
+
+38 items require Group Policy Template in place in order to audit them.
+These items are marked with the label 'CIS_group_policy_template_required' in the respective yml file and are mentioned here:
```

Comment:
I think this comment re: the list below got lost (my fault— I noticed a typo first and ended up adding 2 suggestions for the same line). Let me know your thoughts:

---

Since this information appears to be included in the resolution of each of these policies, I'd reconsider whether we want to duplicate it all.

I think we should remove the list below and change this text to something like:

```suggestion
These items are tagged with the label `CIS_group_policy_template_required` in the YAML file, and details about the required Group Policy templates can be found in each item's `resolution`.
```
Less documentation surface area to maintain, and I don't think the user experience of searching for that tag in the YAML is any worse than going through the list items on this page (as a user I'd want to double-check that the actual file matches what the docs say anyway, been burned too many times 😅 ).